packages/ckeditor5-link/docs/_snippets/features/link-providers.html

```diff
@@ -0,0 +1,9 @@
+<div class="ck ck-content" id="snippet-link-providers">
+	<p>Follow us on social media:</p>
+    <ul>
+        <li>Facebook</li>
+        <li>Twitter</li>
```

Comment:
Wonder when we will stop using this name :D

Reply:
I'm not gonna :D